benchmarking/commons/hpo_main_simulator.py

```diff
@@ -201,65 +207,38 @@ def parse_args(
         else:
             bm_dict = benchmark_definitions
         benchmark_names = list(bm_dict.keys())
-    return args, method_names, benchmark_names, seeds
-
-
-def main(
-    methods: MethodDefinitions,
-    benchmark_definitions: SurrogateBenchmarkDefinitions,
-    extra_args: Optional[ExtraArgsType] = None,
-    map_extra_args: Optional[MapExtraArgsType] = None,
-    post_processing: Optional[PostProcessingType] = None,
-    use_transfer_learning: bool = False,
-):
-    """
-    Runs sequence of experiments with simulator backend sequentially. The loop
-    runs over methods selected from ``methods``, repetitions and benchmarks
-    selected from ``benchmark_definitions``, with the range being controlled by
-    command line arguments.
 
-    :param methods: Dictionary with method constructors
-    :param benchmark_definitions: Definitions of benchmarks
-    :param extra_args: Extra arguments for command line parser. Optional
-    :param map_extra_args: Maps ``args`` returned by :func:`parse_args` to dictionary
-        for extra argument values. Needed if ``extra_args`` given
-    :param post_processing: Called after tuning has finished, passing the tuner
-        as argument. Can be used for postprocessing, such as output or storage
-        of extra information
-    :param use_transfer_learning: If True, we use transfer tuning. Defaults to
-        False
-    """
-    args, method_names, benchmark_names, seeds = parse_args(
-        methods, benchmark_definitions, extra_args
-    )
-    experiment_tag = args.experiment_tag
-    master_random_seed = get_master_random_seed(args.random_seed)
+    method_names = [item for item in methods.keys()]
```

Comment:
`list(methods.keys())`

Reply:
This is important. The use case of also parallelizing over benchmarks is something we would like to have working properly.